samples/bot-types-of-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,136 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send types of cards using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "01/13/2023 05:00:17 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-types-of-cards-nodejs
+
+---
+# Types of Cards
+
+This sample shows the feature where user can send types of cards using bot.
+
+## Interaction with app
+
+![Types Of Cards](Images/TypesOfCards.gif)
+
+## Prerequisites
+
+-  Microsoft Teams is installed and you have an account (not a guest account).
+-  To test locally, [NodeJS](https://nodejs.org/en/download/) must be installed on your development machine (version 16.14.2  or higher).
+-  [ngrok](https://ngrok.com/) or equivalent tunneling solution.
+
+## Setup
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+> the Teams service needs to call into the bot.
+
+1. Register a new application in the [Azure Active Directory – App Registrations](https://go.microsoft.com/fwlink/?linkid=2083908) portal.
+
+2. Setup for Bot
+- Register Azure AD application resource in Azure portal
+- In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+- While registering the bot, use `https://<your_ngrok_url>/api/messages` as the messaging endpoint.
+
+	**NOTE:** When you create your bot you will create an App ID and App password - make sure you keep these for later.
+
+3. Setup NGROK  
+Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http -host-header=rewrite 3978
+    ```
+
+4. Setup for code  
+  - Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  - In a terminal, navigate to `samples/bot-types-of-cards/nodejs`
+
+  - Update the `.env` configuration file for the bot to use the `MicrosoftAppId` and `MicrosoftAppPassword`.  (Note the MicrosoftAppId is the AppId created in step 1 (Setup AAD app registration in your Azure portal), the MicrosoftAppPassword is referred to as the "client secret" in step 1 (Setup for Bot) and you can always create a new client secret anytime.)
```

Comment:
Is this correct or you should use placeholders?

Reply:
fixed